### DISCUSSION TITLE
Mapping 2D->3D RGB-Pointcloud data using python

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#437] [Q&A] [answered]
Hello,

I have O3R225 with VPU (FW 1.0.14). As iVA show live feed of 2D-3D mapping overview. How can i achieve this using python? However i am getting RGB and pointcloud using 2 different ports. But i didnt find any info/example about 2D->3D mapping using ifm3dpy. So where can i find these info/example?

**Comment by lola-masson** [comment, marked as answer]
Hi @Astronomia-maker,

You can find an example here: https://github.com/ifm/documentation/tree/main/SoftwareInterfaces/Toolbox/Registration2d3d

**Reply by Astronomia-maker** [reply]
Thank you @lola-masson
